src/arco_era5/source_data.py

```diff
@@ -559,6 +563,65 @@ def align_coordinates(dataset: xr.Dataset) -> xr.Dataset:
     return dataset
 
 
+def get_pressure_levels_arg(pressure_levels_group: str):
+    return PRESSURE_LEVELS_GROUPS[pressure_levels_group]
+
+
+class LoadTemporalDataForDateDoFn(beam.DoFn):
```

Comment:
Optional: I have a personal preference for making PTransforms instead of using `DoFns`.

Reply:
Making a note of it now. Will update this later as this requires major code changes irrelevant from these.